apps/web/src/store/app.ts

```diff
@@ -22,7 +24,9 @@ export const useAppStore = create<AppState>((set) => ({
   userSigNonce: 0,
   setUserSigNonce: (userSigNonce) => set(() => ({ userSigNonce })),
   isPro: false,
-  setIsPro: (isPro) => set(() => ({ isPro }))
+  setIsPro: (isPro) => set(() => ({ isPro })),
+  showMobileDrawerMenu: false,
+  setShowMobileDrawerMenu: (showMobileDrawerMenu) => set(() => ({ showMobileDrawerMenu }))
```

Comment:
No states for this bls 🙏🏼 

You can check https://flowbite.com/docs/components/drawer/

Reply:
removed states